builder/azure/common/client/tokenprovider_msi.go

```diff
@@ -8,11 +8,11 @@ import (
 // for managed identity auth
 type msiOAuthTokenProvider struct {
 	env      azure.Environment
-	ClientID string
+	clientID string
```

Comment:
I unexported this field because I didn't see any uses of it requiring it be public. Let me know if you want me to yank this change.

Reply:
Makes sense to me!